docs/Accepting_connections_from_Settle.md

```diff
@@ -0,0 +1,32 @@
+## Accepting connections from Settle
+
+!!! IMPORTANT
+    Your API must be exposed the internet.
+
+    Settle no longer uses VPN for securing communication for new integrations.
+
+
+In this scenario, the `Client Service` is a Settle service and the `API Server`
+is your API server.
+
+Settle will provide you with one `service account email` for each Settle service
+that needs to access your server. You must register the email in your system.
+Typically, there will be a single Settle service that will need to be registered.
+
+You will use a GCP Auth Server to verify the token included in the request is valid.
```

Comment:
Should probably mention where the token is found, probably in the HTTP header `Authorization: Bearer <TOKEN>`?

Reply:
:heavy_check_mark: